universal-login-sdk/test/sdk.js

```diff
@@ -103,6 +104,18 @@ describe('SDK - integration', async () => {
       });
     });
 
+    describe('Add keys', async () => {
+      const transactionDetalis = {
+        nonce: 0,
+        gasToken,
+        gasPrice,
+        gasLimit
+      };
+      it('should return execution nonce', async () => {
+        expect(await sdk.addKeys(identityAddress, [otherWallet.address, otherWallet2.address], privateKey, transactionDetalis)).to.eq(4);
```

Comment:
Perhaps a check if keys are added could be useful.

Reply:
I see there is no check in `Add key` as well. This SDK test, it is covered in contracts test, so I think it fine. 
Reviewed ✅